File: PHP/funcionario/src/programa.php
<?php

require __DIR__ . '/../vendor/autoload.php';

use Deivz\Funcionario\entities\Funcionario;

echo("Informe o nome do funcionário: ");
$nome = readline();

echo("Informe o salário do funcionário " . $nome . ": ");
$salario = floatval(readline());

echo("Informe o imposto correspondente ao salário do funcionário " . $nome . ": ");
$imposto = floatval(readline());

$funcionario = new Funcionario($nome, $salario, $imposto);

echo("Funcionário: " . $funcionario. PHP_EOL);

echo("Qual a porcentagem do aumento de salário de " . $nome . "? ");
$salarioAumentado = floatval(readline());
$funcionario->aumentarSalario($salarioAumentado);

echo("Dados atualizados: " . $funcionario);

File: Java/funcionario/src/application/programa.java
package application;

import java.util.Scanner;

import entities.Funcionario;

public class programa {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);

		System.out.println("Informe o nome do funcionário: ");
		String nome = sc.nextLine();
		
		System.out.println("Informe o salário do funcionário " + nome + ": ");
		double salario = sc.nextDouble();
		
		System.out.println("Informe o imposto correspondente ao salário do funcionário " + nome + ": ");
		double imposto = sc.nextDouble();
		
		Funcionario funcionario = new Funcionario(nome, salario, imposto);
		
		System.out.println("Funcionário: " + funcionario);
		
		System.out.println("Qual a porcentagem do aumento de salário de " + nome + ": ");
		double salarioAumentado = sc.nextDouble();
		funcionario.aumentarSalario(salarioAumentado);
		
		System.out.println("Dados atualizados: " + funcionario);
	}

}
